src/const.js

```diff
@@ -87,6 +93,22 @@ export const LOW_LEVEL_MODULES = [
   'system/events', 'sdk/system/events',
 ];
 
+export const DEPRECATED_ENTITIES = [
+  {
```

Comment:
This is causing issues with jscs if I have
```javascript
[{
...
}];
```
Due to conflicting rules. It for some reason jscs requires a comma be after the `{` but not before the `[`.